### DISCUSSION TITLE
I missed my phone number

### DISCUSSION THREAD

**Discussion by Modjaid** [community/community#36121] [Other Feature Feedback, Questions, & Ideas]
Hello, I have a very big problem, I left the country very quickly losing my SIM card. I can’t restore it in any way, how can I bypass account verification, knowing only my mail and password? help my repositories are very dear to me!!!

**Comment by DSDmark** [comment]
So,
There is only one thing you can do, my friend https://support.github.com/
go there and select your choice and send the email of git support https://support.github.com/request/account?tags=hubberfy_authentication
